src/Client/Markup.fs

```diff
@@ -15,17 +15,15 @@ type Clickability =
 
 let goToUrl (e: React.MouseEvent) =
     e.preventDefault()
-    let href = !!e.target?href
+    let href = !!e.currentTarget?href
     Navigation.newUrl href |> List.map (fun f -> f ignore) |> ignore
 
-let viewLinkCentered className page description =
+let viewLinkCentered className page children =
     a [ 
         ClassName className
         Href (toHash page)
         OnClick goToUrl 
-    ] [ 
-        str description
-    ]
+    ] children
```

Comment:
So what are examples of children here?

Reply:
a list of elements (other components, html elems, etc)